README.md

```diff
@@ -12,4 +12,20 @@ n/a
 **Tests**
 n/a
 **UML**
+n/a
+---
+## LAB - Class 02
+**Project:** Classes, Inheritance, Functional  (Notesy)
+**Author:** Jennifer Chinzi
+**Links and Resources**
+n/a
+**Setup**
+npm install minimist
+**How to initialize/run your application (where applicable)**
+e.g. npm start
+**How to use your library (where applicable)**
+n/a
+**Tests**
+npm test
+**UML**
 n/a
```

Comment:
Make sure to leave an extra space on the end of a line so that you don't get a giant paragraph!